packages/nethack4.rb

```diff
@@ -8,10 +8,12 @@ class Nethack4 < Package
   source_sha256 'b143a86b5e1baf55c663ae09c2663b169d265e95ac43154982296a1887d05f15'
 
   binary_url ({
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/nethack4-4.3.0-beta2-1-chromeos-x86_64.tar.xz',
+      i686: 'https://dl.bintray.com/chromebrew/chromebrew/nethack4-4.3.0-beta2-1-chromeos-i686.tar.xz',
+    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/nethack4-4.3.0-beta2-1-chromeos-x86_64.tar.xz',
   })
```

Comment:
Why no ARM packages?

Reply:
It wouldn't build on arm and I tried to fix it but gave up after awhile.  I don't know why I care so much. :)

Reply:
Thanks. I'm working furiously on this cross compilation script right now, and so far it's gone smoothly.

Reply:
Furiously smoothly.  I get it. :)